src/deck_sort.py

```diff
@@ -0,0 +1,40 @@
+"""This will sort a deck of cards"""
+"""['3', '9', 'A', '5', 'T', '8', '2', '4', 'Q', '7', 'J', '6', 'K']"""
+"""['A', '2', '3', '4', '5', '6', '7', '8', '9', 'T', 'J', 'Q', 'K']"""
+
+
+def deck_sort(deck):
+    """This will sort a random deck of cards"""
+    sorted_deck = []
+    idx = 0
+    for card in deck:
+        while True:
+            try:
+                sorted = get_value(sorted_deck[idx])
+                tmp = get_value(card)
+                if sorted >= tmp:
```

Comment:
Avoid using "sorted" as a variable name, as that's Python's built-in sorting function